runelite-client/src/main/java/net/runelite/client/RuneLite.java

```diff
@@ -172,6 +178,8 @@ public static void main(String[] args) throws Exception
 		final OptionParser parser = new OptionParser();
 		parser.accepts("developer-mode", "Enable developer tools");
 		parser.accepts("debug", "Show extra debugging output");
+		parser.accepts("sessionfile", "Use a specified session file").withRequiredArg();
```

Comment:
```suggestion
		ArgumentAcceptingOptionSpec<String>  sessionFileOp = parser.accepts("sessionfile", "Use a specified session file")
		.withRequiredArg()
		.defaultsTo("session");
```
then use you can just do
```
String sessionFile = options.valueOf(sessionFileOp)
```